tvb/simulator/plot/head_plotter.py

```diff
@@ -0,0 +1,136 @@
+from tvb.datatypes.projections import ProjectionMatrix
+
+
+class HeadPlotter(BasePlotter):
+
+    def __init__(self, config=None):
+        super(HeadPlotter, self).__init__(config)
+
+    def _plot_connectivity(self, connectivity, figure_name='Connectivity'):
+        pyplot.figure(figure_name + str(connectivity.number_of_regions), self.config.figures.VERY_LARGE_SIZE)
+        axes = []
+        axes.append(self.plot_regions2regions(connectivity.normalized_weights,
+                                              connectivity.region_labels, 121, "normalised weights"))
+        axes.append(self.plot_regions2regions(connectivity.tract_lengths,
+                                              connectivity.region_labels, 122, "tract lengths"))
+        self._save_figure(None, figure_name.replace(" ", "_").replace("\t", "_"))
+        self._check_show()
+        return pyplot.gcf(), tuple(axes)
+
+    def _plot_connectivity_stats(self, connectivity, figsize=None, figure_name='HeadStats '):
+        if not isinstance(figsize, (list, tuple)):
+            figsize = self.config.figures.VERY_LARGE_SIZE
+        pyplot.figure("Head stats " + str(connectivity.number_of_regions), figsize=figsize)
+        areas_flag = len(connectivity.areas) == len(connectivity.region_labels)
+        axes = []
+        axes.append(self.plot_vector(compute_in_degree(connectivity.normalized_weights), connectivity.region_labels,
+                                     111 + 10 * areas_flag, "w in-degree"))
+        if len(connectivity.areas) == len(connectivity.region_labels):
+            axes.append(self.plot_vector(connectivity.areas, connectivity.region_labels, 122, "region areas"))
+        self._save_figure(None, figure_name.replace(" ", "").replace("\t", ""))
+        self._check_show()
+        return pyplot.gcf(), tuple(axes)
+
+    def _plot_sensors(self, sensors, projection, region_labels, count=1):
+        figure, ax, cax = self._plot_projection(sensors, projection, region_labels,
+                                                title= "%d - %s - Projection" % (count, sensors.sensors_type))
+        count += 1
+        return count, figure, ax, cax
+
+    def _plot_projection(self, sensors, projection, region_labels, figure=None, title="Projection",
+                         show_x_labels=True, show_y_labels=True, x_ticks=numpy.array([]), y_ticks=numpy.array([]),
+                         figsize=None):
+        if not isinstance(figsize, (list, tuple)):
+            figsize = self.config.figures.VERY_LARGE_SIZE
+        if not (isinstance(figure, pyplot.Figure)):
+            figure = pyplot.figure(title, figsize=figsize)
+        ax, cax1 = self._plot_matrix(projection, sensors.labels, region_labels, 111, title,
+                                     show_x_labels, show_y_labels, x_ticks, y_ticks)
+        self._save_figure(None, title)
+        self._check_show()
+        return figure, ax, cax1
+
+    def plot_head(self, head, plot_stats=False, plot_sensors=True):
```

Comment:
This method would need at least model class Head under tvb-library. I do not see it. Is it here?
Alternatively, we could drop this method or change its signature, to receive aggregated DT from a Head as separate params

Reply:
We have chosen to change the method signature to receive Head as separate parameters.